### PR TITLE
New Onboarding: fix monthly pricing in new-launch flow

### LINKED ISSUE #48808: Launch Flow Plans Page: Monthly/Yearly Toggle display issues
#### Steps to reproduce the behaviour

This should reproduce if you're in the correct test flow. These steps could likely be updated to trigger the correct screen via the console.

1. Create a new site on WP.com
2. Launch your site
3. Observe toggle on launch screen

**Screenshot of Launch Flow Plans Screen:**
image

The monthly/yearly toggle in the launch flow above is too close to the header on this screen and it appears as if the toggle component from other screens like this one …

**Example of toggle on Signup screen:**
image

… is appearing on this particular launch flow screen without considering how it would appear on this launch flow screen.

### PR DESCRIPTION
**Pull request by razvanpapadopol**
#### Changes proposed in this Pull Request

* For new-launch flow, move header on top of billing period toggle

#### Testing instructions

* create a site using `/new`
* #48808 should be fixed

Fixes #48808

### CODE REVIEW

File: client/my-sites/plans-features-main/gutenboarding-header/style.scss [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
+	@media ( min-width: 1600px ) {
+		max-width: 1600px;
+		margin-left: -280px;
+	}
+
+	@include breakpoint-deprecated( '<1040px' ) {
+		padding-top: 12px;
+	}
+	// END
+
```
Comment (ciampo) [inline client/my-sites/plans-features-main/gutenboarding-header/style.scss:16]: These changes are a bit hacky and are not really related to the main issue being flagged here — not sure if it's best to include them (and add a follow-up issue/PR to fix those styles properly), or omit these changes completely.

File: client/my-sites/plans-features-main/gutenboarding-header/style.scss [hunk 2]
```diff
@@ -1,18 +1,37 @@
...
 	flex-wrap: wrap;
-	margin: -12px 27px 50px;
 	justify-content: space-between;
 
+	body.is-section-signup .plans-features-main > & > header.formatted-header.is-left-align {
+		margin: 20px;
+	}
+
 	&__select-free-plan {
+		margin: 0 20px;
```
Comment (ciampo) [inline client/my-sites/plans-features-main/gutenboarding-header/style.scss:26]: Same as https://github.com/Automattic/wp-calypso/pull/48811/files#r555959539

**Review by ciampo — COMMENTED** [review]
The main issue seems to be fixed, at least on Chrome on my local machine.

Screenshot 2021-01-12 at 18 49 53

Before merging, I would recommend doing more thorough testing across different browsers (including mobile), and smoke testing around other related signup/launch flows.

I left a couple of comments, but not blockers in case this PR needs to get merged urgently.

File: client/my-sites/plans-features-main/gutenboarding-header/style.scss (hunk 1 above)
Comment (razvanpapadopol) [reply client/my-sites/plans-features-main/gutenboarding-header/style.scss:16]: If we want to [move the header above toggle](https://github.com/Automattic/wp-calypso/pull/48811/files#diff-2545c62593dd002397248377b1c0feb643bd2acc7694054ce094997b1c029aceR432) and keep it aligned with plans grid, hacky is the only way 😬

**Comment by ollierozdarz** [comment]
@ciampo are we able to please address the display issue of the Toggle (assuming it's now below the page header like the screenshot above)? The original styles created by Fillipo were intended to sit on the Blue control background, but now that it's on a White page it's lacking a bounding box/border. See @ianstewart's [comment](https://github.com/Automattic/wp-calypso/issues/48808).

**If possible, it could look this this ([Figma](https://www.figma.com/file/8SZljM999DKC7tWBlONFCR/Launch-Flow-All-users?node-id=1%3A93)):**

Screen Shot 2021-01-13 at 2 19 59 pm

**Review by lsl — APPROVED** [review]
LGTM

**Comment by lsl** [comment]
@ollierozdarz
Screenshot_2021-01-13 Launch your site — WordPress com(2)

Also removed this little border-right accent like thing as it looked out of place (bit above the red line)

Screenshot(11)

**Comment by ollierozdarz** [comment]
@lsl fantastic!

**Comment by lsl** [comment]
This PR isn't the place for it but I think mobile could use a bit of tidy up on this screen, plan info is being pushed out of view for marketing copy.



Something like this maybe?
- Moving "start with free" into the heading subtext as "or start with a free plan." (old launch does the same)
- Shortening the "Getting ready to launch your site" copy down to "Launch your website"
- Center the popover below so it looks less "steppy"

e.g.


Is this worth creating a ticket / mocking in figma?

**Comment by ollierozdarz** [comment]
@lsl this is definitely worth of a ticket in my eyes. This mobile view has been solved previously in the control flow Monthly test, and since this isn't a redesign of that (merely trying to reach parity) we should try and make it as 1:1 as possible while changes where necessary.

**Pull request merged**